src/AbstractDifferentiation.jl

```diff
@@ -57,19 +102,51 @@ function hessian(ab::AbstractBackend, f, x)
     end, x)
 end
 
+
+"""
+    AD.value_and_derivative(ab::AD.AbstractBackend, f, xs::Number...)
+
+Compute the function value `v = f(xs...)` and the derivatives `ds` of `f` wrt the numbers `xs` using the backend `ab`.
+    
+Return a tuple `(v, ds)` where `ds` is a tuple of derivatives, one for each element in `xs`.
```

Comment:
I think this is maybe a bit repetetive, which also makes it a bit less convenient when making changes to e.g. `derivative. Maybe mainly refer to the other docstring?
```suggestion
Return the tuple `(v, ds)` of the function value `v = f(xs...)` and the derivatives `ds = AD.derivative(ab, f, xs...).

See also [`AD.derivative`](@ref).
```